pkg/controller/releasemanager/releasemanager_controller.go

```diff
@@ -374,6 +374,12 @@ func (r *ResourceSyncer) SyncVirtualService() error {
 			r.Instance.UpdateReleaseStatus(releaseStatus)
 		}
 
+		// Wind down retired releases in HPA
+		// TODO(bob): Use some time-based rule to leave 1 instance running for
```

Comment:
I don't think this is necessary. Keeping the release around for a while but turned down to 0 is sufficient.

Reply:
👍